test/e2e/tests/aws/test_batch.py

```diff
@@ -25,17 +25,17 @@
 @pytest.mark.usefixtures("client")
 @pytest.mark.parametrize("api", TEST_APIS)
 def test_batch_api(config: Dict, client: cx.Client, api: str):
-    s3_bucket = config["aws"].get("s3_bucket")
-    if not s3_bucket:
+    s3_path = config["aws"].get("s3_path")
+    if not s3_path:
         pytest.skip(
-            "--s3-bucket option is required to run batch tests (alternatively set the "
+            "--s3-path option is required to run batch tests (alternatively set the "
             "CORTEX_TEST_BATCH_S3_BUCKET_DIR env var) )"
         )
 
     e2e.tests.test_batch_api(
         client,
         api,
-        test_bucket=s3_bucket,
+        test_bucket=s3_path,
```

Comment:
nit: change the function parameter as well

Reply:
Done!